Puzzlepart/did
# Session cookie issues

### Your checklist for this pull request
 
### Review checklist

### Review theme song
🎵  [Specktors - Unz Unz](https://open.spotify.com/track/593Jq3pm0OleC7Mz6STFFk?si=vDdML34JSmqWTJR6ym5EHw) 🎵 

### Description
Set session cookie rolling to false;

See this answer https://stackoverflow.com/a/38287332/
tl;dr:
```
Something to be aware of is certain combinations of these values (along with others)
might produce unexpected behaviour. 
For example, the documentation states:

When this option [rolling] is set to true but the saveUninitialized option is set to false,
the cookie will not be set on a response with an uninitialized session.
```

### How to test
1. Check out locally with [gh](https://github.com/cli/cli)
2. Try to break the login process using incognito mode and multiple tabs at different stages of the login process
3. If you can't provoke an error, this PR might have fixed the issue
